packages/blade/src/components/Icons/index.tsx

```diff
@@ -78,18 +78,231 @@ export { default as TrashIcon } from './TrashIcon';
+export { default as ItalicIcon } from './ItalicIcon';
+export { default as TypeIcon } from './TypeIcon';
+export { default as UnderlineIcon } from './UnderlineIcon';
+export { default as FilmIcon } from './FilmIcon';
+export { default as AirplayIcon } from './AirplayIcon';
+export { default as AtSignIcon } from './AtSignIcon';
+export { default as BluetoothIcon } from './BluetoothIcon';
+export { default as CameraIcon } from './CameraIcon';
+export { default as CameraOffIcon } from './CameraOffIcon';
+export { default as CastIcon } from './CastIcon';
+export { default as MessageCircleIcon } from './MessageCircleIcon';
+export { default as MessageSquareIcon } from './MessageSquareIcon';
+export { default as PhoneCallIcon } from './PhoneCallIcon';
+export { default as PhoneIcon } from './PhoneIcon';
+export { default as PhoneOutgoingIcon } from './PhoneOutgoingIcon';
+export { default as PhoneOffIcon } from './PhoneOffIcon';
+export { default as PhoneMissedIcon } from './PhoneMissedIcon';
+export { default as PhoneIncomingIcon } from './PhoneIncomingIcon';
+export { default as PhoneForwardedIcon } from './PhoneForwardedIcon';
+export { default as RadioIcon } from './RadioIcon';
+export { default as VideoOffIcon } from './VideoOffIcon';
+export { default as VoicemailIcon } from './VoicemailIcon';
+export { default as WifiIcon } from './WifiIcon';
+export { default as WifiOffIcon } from './WifiOffIcon';
+export { default as ApertureIcon } from './ApertureIcon';
+export { default as AwardIcon } from './AwardIcon';
+export { default as BoxIcon } from './BoxIcon';
+export { default as BriefcaseIcon } from './BriefcaseIcon';
+export { default as ChromeIcon } from './ChromeIcon';
+export { default as CircleIcon } from './CircleIcon';
+export { default as CrosshairIcon } from './CrosshairIcon';
+export { default as DiscIcon } from './DiscIcon';
+export { default as DropletIcon } from './DropletIcon';
+export { default as FeatherIcon } from './FeatherIcon';
+export { default as LayersIcon } from './LayersIcon';
+export { default as LayoutIcon } from './LayoutIcon';
+export { default as SidebarIcon } from './SidebarIcon';
+export { default as SquareIcon } from './SquareIcon';
+export { default as TriangleIcon } from './TriangleIcon';
+export { default as LifeBuoyIcon } from './LifeBuoyIcon';
+export { default as LoaderIcon } from './LoaderIcon';
+export { default as OctagonIcon } from './OctagonIcon';
+export { default as PackageIcon } from './PackageIcon';
+export { default as PercentIcon } from './PercentIcon';
+export { default as ShieldIcon } from './ShieldIcon';
+export { default as UmbrellaIcon } from './UmbrellaIcon';
+export { default as WindIcon } from './WindIcon';
+export { default as ArrowUpLeftIcon } from './ArrowUpLeftIcon';
+export { default as ArrowDownRightIcon } from './ArrowDownRightIcon';
+export { default as ArrowDownLeftIcon } from './ArrowDownLeftIcon';
+export { default as ChevronsRightIcon } from './ChevronsRightIcon';
```

Comment:
I think because the icon consists of multiple chevrons 

<img width="106" alt="image" src="https://user-images.githubusercontent.com/35374649/212856016-14fff461-c7db-4691-86cf-d6763fd50dd3.png">

Reply:
Single variants are called appropriately 
<img width="148" alt="image" src="https://user-images.githubusercontent.com/35374649/212856172-958a9086-5b01-4299-9115-b7fc68febefb.png">